code/modules/mob/living/carbon/human/human.dm

```diff
@@ -815,22 +815,22 @@
 			return
 
 		var/list/options = list("Clear"="Clear")
-		for(var/x in subtypesof(/datum/quirk))
+		for(var/x in (subtypesof(/datum/quirk) - /datum/quirk/item_quirk))
```

Comment:
do the same as above where you check for abstract_type instead of slapping this on

Reply:
See other comment.